The Ember Times No. 136 - February 21st 2020

amyrlam opened the pull request:
📝 Ideas, feel free to add to list or claim!

- [ ] Check RFCs
- [ ] Ember Fast CLI v1 release https://twitter.com/vaier/status/1227653780235309058
- [x] New episode of What's New in Ember https://twitter.com/ember_map/status/1228412177775841287 (🔏 @jessica-jordan)
- [x] Tomsweeper (Minesweeper created with Octane) by scudco (🔒 @ijlee2)
  - https://discordapp.com/channels/480462759797063690/487382592841187328/678062355502071808
- [x] Ember 3.16 https://blog.emberjs.com/2020/02/12/ember-3-16-released.html (🔒 @amyrlam)
- [ ] Evolution of Ember at Intercom https://www.intercom.com/blog/videos/evolution-of-ember-at-intercom/
- [x] Request for videos of people talking about how Ember changed them as a developer in the next week or two (for EmberConf) (:lock: @wycats) 
- [x] I published my first Ember Addon https://dev.to/xiwcx/i-published-my-first-ember-addon-138c (🔒 @chrisrng )
- [ ] Using WebAssembly in Ember.js https://mfeckie.dev/wasm-in-ember/
- [x] Ember Survey is set to be sent out, calling all participants (🔒 @jaredgalanis)
- [ ] RFC: `{{set}}` helper https://github.com/emberjs/rfcs/pull/594

### Outstanding Team Quotes about Octane from https://github.com/ember-learn/ember-blog/pull/459

- [ ] @MelSumner
- [ ] @kategengler
- [ ] @mixonic
- [ ] @pzuraq

- [ ] More octane at company x stories? Square, others?

📆 Future stories

- [ ] **UPDATE** _this should go out in the 2/28 issue per 2/20 learning team meeting_ Write up on current website update status. Call for contributors, explaining the styleguide and which projects contributors can help with. Ping @mansona when this topic is due for release in the Times 

🤔 Questions?

See #support-ember-times on Discord or CONTRIBUTING.

✅ Todos:

- Check https://github.com/ember-learn/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos) for ideas
- Use https://github.com/ember-learn/emberjs-times-tools README
- Check Netlify preview, to ensure display on emberjs.com/blog is good
  - [ ] Check buttons
- [ ] Crosspost on dev.to, delete Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
  - [ ] post $natural

amyrlam committed:
Set up new issue

amyrlam committed:
Set up new issue

xiwcx commented:
Hi, not sure if this is the place, but I wrote an article on writing my first addon that might be good for the newsletter: https://dev.to/xiwcx/i-published-my-first-ember-addon-138c

Thanks.

amyrlam commented:
@xiwcx 🙏 This is the right place! Thank you, just added it to the list above! 

In addition to the issue PR, you can also find us in [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on the [Ember Discord](https://discordapp.com/invite/zT3asNS).

amyrlam committed:
Add blurb on Ember 3.16

jayjayjpg committed:
Merge pull request #535 from ember-learn/amy/embertimes-136

Add blurb on Ember 3.16

chrisrng committed:
blog(embertimes136): ember-fullcalendar

amyrlam committed:
🚨 Fix alex

✏️ Also, minor formatting fix

amyrlam committed:
Merge branch 'blog/embertimes-136' into chrisrng-patch-1

amyrlam committed:
Add link to dev.to post in body text

Sometimes people don't notice that the header is clickable - maybe we can talk about tweaking this design @ EmberConf!

amyrlam committed:
Merge pull request #537 from ember-learn/chrisrng-patch-1

blog(embertimes136): ember-fullcalendar

amyrlam committed:
Add draft of Yehuda's post

Co-authored-by: Yehuda Katz <wycats@gmail.com>

amyrlam committed:
🚨 Fix alex linter

amyrlam committed:
🚨 Update markdownlint

amyrlam committed:
Update based on Airtable changes

re: https://airtable.com/shrYpEeT3xtyst5Gq

ijlee2 committed:
Covered release of Tomsweeper

ijlee2 committed:
Merge pull request #539 from ijlee2/blog/embertimes-136

Covered release of Tomsweeper

jaredgalanis committed:
adds 2020 ember community survey

jaredgalanis committed:
Merge branch 'blog/embertimes-136' into ember-community-survey-2020

jaredgalanis committed:
typo fix

jaredgalanis committed:
some updates

jaredgalanis committed:
Merge pull request #540 from ember-learn/ember-community-survey-2020

Ember community survey 2020

jayjayjpg committed:
Merge branch 'blog/embertimes-136' into amyrlam-patch-1

jayjayjpg committed:
Merge pull request #538 from ember-learn/amyrlam-patch-1

Add draft of Yehuda's post

jayjayjpg committed:
blog(embertimes#136): add writeup on embermap el. mod. forw. video

jayjayjpg committed:
Update source/2020-02-21-the-ember-times-issue-136.md

jayjayjpg committed:
Update source/2020-02-21-the-ember-times-issue-136.md

Co-Authored-By: Isaac Lee <16869656+ijlee2@users.noreply.github.com>

jayjayjpg committed:
Update source/2020-02-21-the-ember-times-issue-136.md

Co-Authored-By: Isaac Lee <16869656+ijlee2@users.noreply.github.com>

jayjayjpg committed:
Update source/2020-02-21-the-ember-times-issue-136.md

Co-Authored-By: Isaac Lee <16869656+ijlee2@users.noreply.github.com>

jayjayjpg committed:
Merge pull request #541 from jessica-jordan/embertimes-jj

blog(embertimes#136): add writeup on embermap el. mod. forw. video

jayjayjpg committed:
blog(embertimes#136): copy edits

jayjayjpg reviewed (approved):
This reads great! Thank you for your writing, editing, triaging and marketing efforts everyone :tada:
<img width="975" alt="Screenshot 2020-02-21 at 21 04 33" src="https://user-images.githubusercontent.com/8811742/75067640-c78b5100-54ed-11ea-8c0a-82509f53c3f5.png">
